prelude/__internal__/Wish.mad

```diff
@@ -8,6 +8,12 @@ of :: a -> Wish e a
 export of = (a) => (Wish((bad, good) => (good(a))))
 
 
+bad :: e -> Wish e a
+export bad = (e) => (
+  Wish((bad, good) => (bad(e)))
```

Comment:
We should add a rule for not clobbering / shadowing variables in the upper scope (here, the interior `bad` should be invalid).

```
export bad = (e) => (
  Wish(
    (fail, _) => ( fail(e) )
  )
)
```